spec/classes/init_spec.rb

```diff
@@ -268,15 +266,13 @@
         :packages           => [ 'libpam0g', ],
         :files              => [
           { :prefix         => 'pam_common_',
-            :types          => ['auth', 'account', 'password', 'session', 'noninteractive_session' ],
+            :types          => ['auth', 'account', 'password', 'session', 'session_noninteractive' ],
```

Comment:
this would change the name of the file and i believe would result in both `noninteractive_session` and `session_noninteractive` being on disk. Does that pose any issue?

Reply:
I double checked the code that it only change the internal resource name but leaves the path untouched.

There was a special line (394) needed for these cases in the spec tests:
https://github.com/ghoneycutt/puppet-module-pam/blob/v2.33.0/spec/classes/init_spec.rb#L389-L403. That can now get removed again :)